.github/workflows/deploy-test.yml

```
@@ -0,0 +1,90 @@
+name: CI/CD - Searchweb (version.txt 기반)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)